### DISCUSSION TITLE
在更换新电脑后重新获取PCL2内测版（更新+解锁码） 更新完成后PCL2会闪退且文件消失

### DISCUSSION THREAD

**Discussion by AptS-1547** [Meloong-Git/PCL#4379] [提问] [answered]
如题，今天在给新电脑装上PCL2时
在输入更新密钥完成更新后 会出现闪退，且文件消失
我猜想有可能是我触动了反盗版分发的代码0.0 （使用火绒安全软件，且推出安全软件后问题未恢复）
请问龙猫大大这种情况该怎么解决，谢谢

**Comment by allMagicNB** [comment, marked as answer]
一眼被 Windows Defender 拦了……

**Comment by AptS-1547** [comment]
已解决，傻逼Defender干的，麻烦了
